test/Microsoft.AspNetCore.Razor.Runtime.Test/Runtime/TagHelpers/TagHelperExecutionContextTest.cs

```diff
@@ -289,7 +289,7 @@ public void HtmlAttributes_IgnoresCase(string originalName, string updatedName)
         {
             // Arrange
             var executionContext = new TagHelperExecutionContext("p", TagMode.StartTagAndEndTag);
-            executionContext.HTMLAttributes[originalName] = "hello";
+            executionContext.AddHtmlAttribute(originalName, "hello");
```

Comment:
This test slightly overlaps the changes dont [here](https://github.com/aspnet/Razor/pull/683). I split the changes after doing both and created two PRs. In this situation I call AddHtmlAttribute in order to instantiate the underlying `executionContext.HtmlAttributes` list. Otherwise it `null` refs.